src/components/tree-view/tree-item.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
```

Comment:
lets rename this file to `default-tree-item`

Reply:
done